packages/manager/modules/pci/src/components/project/empty/index.js

```diff
@@ -0,0 +1,14 @@
+import angular from 'angular';
+
+import component from './empty.component';
+
+import './index.less';
+
+const moduleName = 'ovhManagerPciComponentsProjectEmpty';
+
+angular
+  .module(moduleName, [
```

Comment:
Since you are using `translate` directive, I think you have to declare a dependency to `angular-translate` module. Isn't it?